bin/grocery-delivery

```diff
@@ -254,7 +325,9 @@ GroceryDelivery::Hooks.prerun(GroceryDelivery::Config.dry_run)
 
 if repo.exists?
   action('Updating repo')
-  repo.update unless GroceryDelivery::Config.dry_run
+  unless GroceryDelivery::Config.dry_run
+    repo.update if GroceryDelivery::Config.repo_update
+  end
```

Comment:
This is fine, but it's a bit awkward. I would have done
```suggestion
  if GroceryDelivery::Config.repo_update && !GroceryDelivery::Config.dry_run
    repo.update
  end
```

Still < 80 chars. :)

Reply:
Good call. That is better. Commited.